### PR TITLE
DM-44256: Add DDV Flutter app to RubinTV

### PR DESCRIPTION
**Pull request by ugyballoons**
Flutter installation via GHA CI.
Flutter run against project via start daemon.
Flutter app mounted at rubintv/ddv/app.
Websockets between Flutter app and worker pods handled in FastAPI.

### CODE REVIEW

File: python/lsst/ts/rubintv/main.py [hunk 1]
```diff
@@ -97,6 +98,7 @@ async def add_cache_control_header(
         response.headers["Cache-Control"] = "public, max-age=3600, must-revalidate"
         return response
 
+<<<<<<< HEAD
```
Comment (fred3m) [inline python/lsst/ts/rubintv/main.py:101]: Looks like you have an incomplete rebase/merge conflict to resolve.

File: start-daemon.sh [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+# inititiate Flutter app
+if ! command -v flutter &> /dev/null
+then
+    echo "Error: Flutter not installed" >&2
+else
+    echo -e "ADDRESS=<hostname>\nPORT=8080" > .env
+    flutter build web
+    cp build/web ddv
+fi
+
```
Comment (fred3m) [inline start-daemon.sh:12]: I'm not totally sure that this script is correct. `flutter build web` (as well as the following line) needs to be run from the `rubintv_visualization` directory. So you'll have to `cd` to that directory, run the install commands, and `cd` back into the main path (which also means chaning `cp build/web ddv` to `cp build/web ../ddv`.

**Review by fred3m — CHANGES_REQUESTED** [review]
This looks good except for a few things. 
1. Could you please add docstrings (my original code was documented, so you should be able to mostly copy/paste those)?
2. Make sure to fix the incomplete rebase
3. Double check the install script.

File: python/lsst/ts/rubintv/main.py (hunk 1 above)
Comment (ugyballoons) [reply python/lsst/ts/rubintv/main.py:101]: Well, that's embarassing! I don't how this even got past pre-commit... 🙄

**Review by fred3m — APPROVED** [review]
LGTM, just make sure you go back to the docstrings and update them. You changed some of the function signatures so they don't match right now.

**Pull request merged**